docs/index.md

```diff
@@ -202,6 +202,7 @@ ViewComponent is built by over a hundred members of the community, including:
 * [Shogun](https://getshogun.com/)
 * [Cometeer](https://cometeer.com/)
 * [Framework](https://frame.work/)
+* [Brightline](https://hellobrightline.com)
```

Comment:
Can you reorder the list alphabetically please?

We [talked about it](https://github.com/github/view_component/pull/1287#issuecomment-1049451765) previously but did not do it yet.

Reply:
Sure thing!

Reply:
@Spone I'm not sure my CHANGELOG change was in the right place. Can you confirm?

Reply:
Please move it to the top, just below the `main` heading :)